platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -24,24 +24,36 @@
  * This bundle implements views of image telemetry.
  * @namespace platform/features/imagery
  */
+
 define(
-    ['moment'],
-    function (moment) {
+    [
+        'zepto',
+        'lodash'
+    ],
+    function ($, _) {
 
         /**
          * Controller for the "Imagery" view of a domain object which
          * provides image telemetry.
          * @constructor
          * @memberof platform/features/imagery
          */
-        function ImageryController($scope, openmct) {
+
+        function ImageryController($scope, $window, openmct) {
             this.$scope = $scope;
+            this.$window = $window;
             this.openmct = openmct;
             this.date = "";
             this.time = "";
             this.zone = "";
             this.imageUrl = "";
+            this.requestCount = 0;
+            this.lastBound = undefined;
+            this.autoScroll = false;
+            this.scrollable =
```

Comment:
thanks for mentioning this in person-- you should depend on $element and then wrap that with zepto to attach your handler.  Otherwise this fails to work when you're in a layout with multiple image history views.